### DISCUSSION TITLE
Generic RSS article titles

### DISCUSSION THREAD

**Discussion by mcandre** [community/community#116799] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I subscribe to GitHub repo RSS feeds in order to be notified automatically for new versions.

However, the titles don't appear to include the name of the repository. So my RSS reader / email inbox gets confusing fast. "3.0" is meaningless without a clear indication of which software component has that release.

Please prefix RSS feed article titles with the repository name, in order to disambiguate with related feeds for other repositories.

**Comment by giovannivicentin** [comment]
Hi mcandre,

I completely understand how the lack of repository names in the RSS feed titles can lead to confusion, especially when you're monitoring multiple repositories for new versions. It's essential to have clear and informative titles in the feeds to quickly identify updates without having to delve into each one individually.

While GitHub's default RSS feed behavior might not include the repository names in the titles, there are a couple of workarounds you might consider:

1. **Custom RSS Feed Generator**: There are third-party services and tools available that can generate custom RSS feeds with more detailed titles. These services often allow you to specify the format of the feed titles, including the repository name, release number, and other relevant information.

2. **RSS Feed Enhancer Extensions**: Depending on the RSS reader you use, there might be browser extensions or plugins available that can enhance RSS feeds by adding additional information like repository names to the titles. Some RSS readers might even offer this functionality built-in, so it's worth exploring the settings of your current reader.

3. **Scripting Solution**: If you're comfortable with scripting, you could write a small script to fetch the GitHub repository releases and generate a custom RSS feed with the desired title format. This approach offers the most flexibility, as you can tailor the feed to include exactly what you need.

I also recommend providing this feedback directly to GitHub if you haven't already. GitHub continuously evolves based on user feedback, and expressing your need for clearer RSS feed titles could influence future updates to the platform.

In the meantime, if you decide to go the route of a custom solution and need any guidance, please don't hesitate to reach out. I'm here to help!

Best,
Giovanni Vicentin
